testing/sleep10.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 """
 This script implements a sleep (in Linux) and timeout (in Windows) command
-using Python for 3 seconds.
+using Python for 8 seconds.
```

Comment:
@dhblum I guess this is 10 seconds as well.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/62/commits/d46b08bf33e6fad40d0f4f67c6107777425c1ce8.